client-apis/decrypt.md

```diff
@@ -161,7 +163,8 @@ that implementation SHOULD NOT provide an API that allows the caller to stream t
 ### Encryption Context
 
 The [encryption context](../framework/structures.md#encryption-context) that is used as
-additional authenticated data during the decryption of the input [encrypted message](#encrypted-message).
+additional authenticated data during the decryption of the input [encrypted message](#encrypted-message) and
+the [encryption context for authentication only](../client-apis/encrypt.md#encryption-context-not-stored-in-the-message) if available.
```

Comment:
It's all used as AAD during the decryption, so just appending an "and ..." here doesn't make sense.
```suggestion
additional authenticated data during the decryption of the input [encrypted message](#encrypted-message).
Specifically, the union of the encryption context serialized into the message header and
the encryption context for authentication only, if available.
```

Reply:
Sprinkle a MUST in there as well, so it is easier to duvet.